docs/src/deployment/traffic.rst

```diff
@@ -171,3 +171,49 @@ the validator app will
 
 For configuring the built-in top-up automation, please refer to the :ref:`validator deployment guide <helm_validator_topup>`.
 Configuring alternative methods for buying traffic, e.g., using third-party services, exceeds the scope of this documentation.
+
+.. _traffic_wasted:
+
+Wasted traffic
+--------------
+
+`Wasted traffic` is defined as synchronizer events that have been sequenced but will not be delivered to their recipients.
+For validators, which are subject to traffic fees,
+wasted traffic implies that :ref:`traffic <traffic_accounting>` has been charged for a message that was ultimately not delivered.
+Not all failed submissions result in wasted traffic:
+wasted traffic only occurs whenever a synchronizer event is rejected after sequencing but before delivery.
+Some level of wasted traffic is expected and unavoidable, due to factors such as:
+
+- Submission request amplification.
+  Participants that use BFT sequencer connections duplicate submission requests after a timeout to ensure speedy delivery in the face of nonresponsive sequencers;
+  if processing was simply slower than usual but the sequencer was not faulty, the duplicate request counts as wasted traffic.
+- Duplication of messages within the ordering layer, typically linked to transient networking issues or load spikes.
+- Duplication of submissions on the participant/app side, for example when catching up after restoring from a backup or after some crashes.
+
+Validator perspective
++++++++++++++++++++++
+
+Validator operators are encouraged to monitor the rate of failed submissions on their validators and investigate the causes of repeatedly failing submissions eagerly.
```

Comment:
Do we document how they can do so? It feels a bit weird to talk about monitoring failed submissions but then we only tell people how to monitor wasted traffic after you explain that not all failed submisions are wasted traffic